projects/packages/connection/src/class-xmlrpc-connector.php

```diff
@@ -7,6 +7,8 @@
 
 namespace Automattic\Jetpack\Connection;
 
+use IXR_Error;
```

Comment:
If the class doesn't exist (presumably a scenario that happens given line 74), would this throw an error?

Also, is there an actual benefit to using `use` in this case? Or is this just to satisfy Phan?

Reply:
No, `use` just creates an alias name for the class in this file. It doesn't care whether the named class actually exists.

We could do `\IXR_Error` everwhere instead, but IMO it tends to be a bit cleaner to not have the backslashes all throughout the code when it's used more than once. Especially in phpdocs.